### DISCUSSION TITLE
How long does it take after approval?

### DISCUSSION THREAD

**Discussion by minjunee98** [community/community#145903] [GitHub Education]
### Hi everyone,

It’s been more than 3 days since my approval, and I still haven’t received the Student Developer Pack. 
How much longer should I wait?

**Comment by LastwhispersLee** [comment]
I applied for it on November 28th, and it is still in the status of “Your request was approved. Your benefits will be available soon.” I'm already getting anxious

**Reply by LastwhispersLee** [reply]
How can we let the Education community team see it?

**Reply by DogukanTopcu** [reply]
Same problem, I applied on 28 November. It still in the status of "Your request was approved. Your benefits will be available soon."

**Reply by pal3dreamer** [reply]
I applied on 27th november, still haven't received the student developer pack :(

**Reply by ttorhan** [reply]
same for me, I would like to access and try out the features as soon as possible. Hope someone review all and approve.

**Reply by tianyindique** [reply]
I have the same problem.My request was approved on 27th november, but I haven't recieved the student developer pack untill now. 
Why is GitHub so slow, while JetBrains' application and entitlement distribution can be completed immediately.I'm really confused, I can't wait anymore.

**Reply by LastwhispersLee** [reply]
Fortunately, I just received an email from github education stating that I have been granted student benefits. Good luck to you all!

**Reply by tianyindique** [reply]
> Fortunately, I just received an email from github education stating that I have been granted student benefits. Good luck to you all!

I envy you😭

**Reply by ttorhan** [reply]
I have been granted student benefits. Good luck to you all!

**Reply by KotraHaridutt** [reply]
My request got approved on November 29th.. And did not receive the mail.. Waiting for the benefits to access and to explore and all the features.. Thank you

**Reply by tianyindique** [reply]
I just checked my email. It's great to see that I have received the benefits🥰. Good luck to you all!
![image](https://github.com/user-attachments/assets/e17b8084-73c5-4776-8aa5-4a38a2eee114)

**Comment by davevad93** [comment]
Hi @minjunee98 ,

What's the status of your application?

If the label is green and states "`Congrats! Your request for @minjunee98 was approved and your benefits will be available soon.` that means you have to wait. It usually takes a few days, but sometimes it can take a little longer, even a couple of weeks; the waiting time may vary depending on different factors, such as the volume of requests.

If the label is purple and states "`Congrats! Your request for @minjunee98 was approved and your benefits are now available.`", that means you should have access to your benefits. Try first going to the [pack offers page](https://education.github.com/pack), scroll down to the "***All offers***" section and select one of the available offers, for example [Copilot](https://education.github.com/pack/redeem/copilot-student), if you can redeem it you are fine, if you can't, I suggest you to raise a ticket [here](https://support.github.com/contact/education), select "`I am a verified student having trouble redeeming my Global Campus offers`" and explain your issue.

For further reference, take a look at the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352):

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

And also the following discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268, #140287
